module-sys/README.md

```diff
@@ -57,22 +64,42 @@ Please check "Caveats & good practices" chapter for more info
 
 Timers are system feature and is Service/Application thread safe. Timer callback life expectancy should be as short as possible for system to work smoothly (as with any other message call)
 
+**WARNING:** Do not use FreeRTOS timers, without utmost care it can cause data races and obstruct sys::Timres and system in generall.
```

Comment:
```suggestion
**WARNING:** Do not use FreeRTOS timers, without utmost care it can cause data races and obstruct sys::Timres and system in general.
```
```suggestion
**WARNING:** Do not use FreeRTOS timers, without utmost care it can cause data races and obstruct sys::Timres and system in generall.
```